actions/bind

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/python3
+# -*- mode: python -*-
+#
+# This file is part of Plinth.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+"""
+Configuration helper for BIND server.
+"""
+
+import argparse
+
+from plinth import action_utils
+
+
+CONFIG_FILE = '/etc/bind/named.conf.options'
+
+value1 = 'acl goodclients { \n   localnets;\n};\n'
+value2 = '        recursion yes;\n           allow-query { goodclients; };\n\n'
+value3 = '	// 	8.8.8.8;\n	// 	8.8.4.4;\n'
+value4 = '	//forward first;\n'
+value5 = '  //dnssec-enable yes;\n'
+
+def parse_arguments():
+    """Return parsed command line arguments as dictionary"""
+    parser = argparse.ArgumentParser()
+    subparsers = parser.add_subparsers(dest='subcommand', help='Sub command')
+    subparsers.add_parser('setup', help='Setup for BIND')
+
+    configure = subparsers.add_parser('configure', help='Configure BIND')
+    configure.add_argument('--set-forwarding', choices=['true', 'false'],
+                           help='Set forwarding true/false')
+    configure.add_argument('--enable-dnssec', choices=['true', 'false'],
+                           help='Set DNSSEC true/false')
+
+    dns = subparsers.add_parser('dns', help='Add DNS')
+    dns.add_argument('--add', help='Add DNS server')
```

Comment:
The form would show the current setting for DNS forwarding servers. I think we can just have servers separated by a space, so "8.8.8.8 8.8.4.4" would be shown as the default. If the field is changed when the form is submitted, then we overwrite whatever is in the config file, with the new setting. (Perhaps could do some basic validation on the form field as well.)